frontend/express/public/localization/dashboard/dashboard.properties

```diff
@@ -219,10 +213,6 @@ common.confirm = Confirm
 common.remove = Remove
 common.next = Next
 common.previous = Previous
-common.select-at-least = At least {0} items should be selected. 
-common.select-at-most = At most {0} items can be selected.
-common.and = and
```

Comment:
Hmm, I agree with Anna. It is weird though... I don't think that you explicitly removed these fields, so why are they deleted?

Reply:
not sure, solving rn.